app/bundles/CoreBundle/Helper/BuilderTokenHelper.php

```diff
@@ -11,37 +11,61 @@
 
 namespace Mautic\CoreBundle\Helper;
 
+use Doctrine\DBAL\Connection;
 use Doctrine\DBAL\Query\Expression\CompositeExpression;
-use Mautic\CoreBundle\Factory\MauticFactory;
+use Doctrine\ORM\EntityManager;
+use Mautic\CoreBundle\Security\Permissions\CorePermissions;
 
 /**
  * Class BuilderTokenHelper.
  */
 class BuilderTokenHelper
 {
-    protected $viewPermissionBase;
-    protected $modelName;
-    protected $langVar;
-    protected $bundleName;
+    private $isConfigured = false;
+
+    private $security;
+    private $entityManager;
+    private $connection;
+    private $userHelper;
+
     protected $permissionSet;
+    protected $modelName;
+    protected $viewPermissionBase = null;
+    protected $langVar            = null;
+    protected $bundleName         = null;
```

Comment:
What's the reason to assign null here? Isn't it the same as leaving it without it?